bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/repeater.js

```diff
@@ -18,6 +18,7 @@ export default () => [
   pn('rangeStep', 'Range Step', 'Step of range (for "range" source type)'),
   pi('groupItem', 'Group Item', 'Group item to whose members will be iterated (for "itemsInGroup" source type)'),
   pt('itemTags', 'Item Tags', 'Iterate over items with the given tags (comma-separated, for "itemsWithTags" source type)'),
+  pt('ruleTags', 'Rule Tags', 'Iterate over rules with the given tags (comma-separated, for "rulesWithTags" source type)'),
```

Comment:
You need to add the `rulesWithTags` source type to the options array above in line 12.
Remember to regenerate the component docs after this.